packages/core/test/unit/query-generator/create-table-query.test.ts

```diff
@@ -0,0 +1,342 @@
+    });
+  });
+
+  it('produces a CREATE TABLE query with two VARCHAR attributes and a tableName object', () => {
+    expectsql(queryGenerator.createTableQuery({ tableName: 'myTable', schema: 'mySchema' }, { title: 'VARCHAR(255)', name: 'VARCHAR(255)' }), {
+      default: 'CREATE TABLE IF NOT EXISTS [mySchema].[myTable] ([title] VARCHAR(255), [name] VARCHAR(255));',
+      'mariadb mysql': 'CREATE TABLE IF NOT EXISTS `mySchema`.`myTable` (`title` VARCHAR(255), `name` VARCHAR(255)) ENGINE=InnoDB;',
+      mssql: `IF OBJECT_ID(N'[mySchema].[myTable]', 'U') IS NULL CREATE TABLE [mySchema].[myTable] ([title] VARCHAR(255), [name] VARCHAR(255));`,
+      sqlite: 'CREATE TABLE IF NOT EXISTS `mySchema.myTable` (`title` VARCHAR(255), `name` VARCHAR(255));',
+      ibmi: `BEGIN DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42710' BEGIN END; CREATE TABLE "mySchema"."myTable" ("title" VARCHAR(255), "name" VARCHAR(255)); END`,
+    });
+  });
+
+  it('produces a CREATE TABLE query with three different INT attributes', () => {
+    expectsql(queryGenerator.createTableQuery('myTable', { int: 'INTEGER', bigint: 'BIGINT', smallint: 'SMALLINT' }), {
+      default: 'CREATE TABLE IF NOT EXISTS [myTable] ([int] INTEGER, [bigint] BIGINT, [smallint] SMALLINT);',
+      'mariadb mysql': 'CREATE TABLE IF NOT EXISTS `myTable` (`int` INTEGER, `bigint` BIGINT, `smallint` SMALLINT) ENGINE=InnoDB;',
+      mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([int] INTEGER, [bigint] BIGINT, [smallint] SMALLINT);`,
+      ibmi: `BEGIN DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42710' BEGIN END; CREATE TABLE "myTable" ("int" INTEGER, "bigint" BIGINT, "smallint" SMALLINT); END`,
+    });
+  });
+
+  it('produces a CREATE TABLE query with three different INT SERIAL attributes', () => {
+    expectsql(queryGenerator.createTableQuery('myTable', { serial: 'INTEGER SERIAL', bigserial: 'BIGINT SERIAL', smallserial: 'SMALLINT SERIAL' }), {
+      default: 'CREATE TABLE IF NOT EXISTS [myTable] ([serial] INTEGER SERIAL, [bigserial] BIGINT SERIAL, [smallserial] SMALLINT SERIAL);',
+      'mariadb mysql': 'CREATE TABLE IF NOT EXISTS `myTable` (`serial` INTEGER SERIAL, `bigserial` BIGINT SERIAL, `smallserial` SMALLINT SERIAL) ENGINE=InnoDB;',
+      postgres: 'CREATE TABLE IF NOT EXISTS "myTable" ("serial" SERIAL, "bigserial" BIGSERIAL, "smallserial" SMALLSERIAL);',
+      mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([serial] INTEGER SERIAL, [bigserial] BIGINT SERIAL, [smallserial] SMALLINT SERIAL);`,
+      ibmi: `BEGIN DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42710' BEGIN END; CREATE TABLE "myTable" ("serial" INTEGER SERIAL, "bigserial" BIGINT SERIAL, "smallserial" SMALLINT SERIAL); END`,
+    });
+  });
+
+  it('produces a CREATE TABLE query with two INTEGER attributes with comments', () => {
```

Comment:
I don't think the integer part is relevant, what seems to be tested is the comment part

The attribute could be reduced to simplify the test